### PR TITLE
[T2A2][T16-A1] ZhaoDingfan

### PR DESCRIPTION
**Pull request by dfz2019**

### CODE REVIEW

**Comment by dfz2019** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -514,7 +514,17 @@ private static int extractTargetIndexFromDeletePersonArgs(String rawArgs) {
      * @return whether it is valid
      */
     private static boolean isDisplayIndexValidForLastPersonListingView(int index) {
-        return index >= DISPLAYED_INDEX_OFFSET && index < getLatestPersonListingView().size() + DISPLAYED_INDEX_OFFSET;
+        //return index >= DISPLAYED_INDEX_OFFSET && index < getLatestPersonListingView().size() + DISPLAYED_INDEX_OFFSET;
```
Comment (PierceAndy) [inline src/seedu/addressbook/AddressBook.java:517]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

**Comment by PierceAndy** [comment]
Try not to accumulate multiple unrelated changes in one commit. Have a single commit per significant refactor.  6593122d4ad1b36229b79248daeb6dec35b041ef should have been three separate commits.

**Comment by PierceAndy** [comment]
@ZhaoDingfan Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**